cadquery/occ_impl/shapes.py

```diff
@@ -631,6 +631,22 @@ def _center_of_mass(shape: "Shape") -> Vector:
 
         return Vector(Properties.CentreOfMass())
 
+    @staticmethod
+    def matrixOfInertia(obj: "Shape") -> List[List[float]]:
+        """
+        Calculates the matrix of inertia of an object.
+        :param obj: Compute the matrix of inertia of this object
+        """
+        Properties = GProp_GProps()
+        calc_function = shape_properties_LUT[shapetype(obj.wrapped)]
+
+        if calc_function:
+            calc_function(obj.wrapped, Properties)
+            moi = Properties.MatrixOfInertia()
+            return [[moi.Value(i, j) for j in range(1, 4)] for i in range(1, 4)]
+
+        raise NotImplementedError
```

Comment:
It seems like this is not covered. Under what circumstances would this line be reached? Could you extend the test?

Reply:
A test has been added to cover the `NotImplementedError` on `Vertex`